package.json

```diff
@@ -60,7 +60,7 @@
   "dependencies": {
     "@datadog/native-appsec": "^1.2.1",
     "@datadog/native-iast-rewriter": "^0.1.0-beta.3",
-    "@datadog/native-iast-taint-tracking": "^0.1.0-beta.1",
+    "@datadog/native-iast-taint-tracking": "^0.1.0-beta.2",
```

Comment:
Just to make sure, I looked quickly at the library and it's unclear if it bundles the prebuilds in the module on npm directly, it seems to upload the artifacts elsewhere. It also doesn't seem to get rid of the `binding.gyp` which has historically caused issues for users when a local build is triggered unexpectedly.